### PR TITLE
New workflow conversion and validation tool

### PR DESCRIPTION
**Pull request by teo**

### CODE REVIEW

File: walnut/schemata/Task.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/obviyus/Controlss/walnut/schemata/Task.json",
```
Comment (miltalex) [inline walnut/schemata/Task.json:3]: Shouldn't the `$id` be an absolute-URI in a domain which we control? (e.g `https://github.com/AliceO2Group/Control/walnut/schemata/Task.json`)

Comment (teo) [reply walnut/schemata/Task.json:3]: Ultimately I question the need of an ID at all, but if we do define one, what you propose seems correct.

File: walnut/schemata/Task.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                    },
+                    "rateLogging": {
+                        "title": "Rate Logging",
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "name",
+                    "type",
+                    "target"
```
Comment (miltalex) [inline walnut/schemata/Task.json:86]: Not always present see [stfb-internal-dpl-global-binary-file-sink.yaml](https://github.com/AliceO2Group/ControlWorkflows/blob/flp-suite-v0.10.0/tasks/stfb-internal-dpl-global-binary-file-sink.yaml)

Comment (teo) [reply walnut/schemata/Task.json:86]: You're right, `target` can be omitted.

**Review by miltalex — DISMISSED** [review]

**Review by miltalex — APPROVED** [review]

**Pull request merged**